src/examples/src/paymentsRefunds/create/create.ts

```diff
@@ -0,0 +1,19 @@
+import MercadoPago, { PaymentsRefunds } from '@src/index';
+
+/**
+ * Mercado Pago Refund.
+ *
+ * @see {@link https://www.mercadopago.com.br/developers/pt/reference/chargebacks/_payments_id_refunds/post Documentation }.
```

Comment:
bruna, vamos colocar a documentação sempre em inglês, é só mudar de `pt` para `en`
```suggestion
 * @see {@link https://www.mercadopago.com.br/developers/en/reference/chargebacks/_payments_id_refunds/post Documentation }.
```

da uma checada nos outros exemplos, existem mais casos

Reply:
Acrescentando a esse ponto, acredito que deveríamos deixar somente `https://www.mercadopago.com`

Reply:
Feito!